src/rules/trailingCommaRule.ts

```diff
@@ -86,21 +187,21 @@ class TrailingCommaWalker extends Lint.AbstractWalker<Options> {
                 case ts.SyntaxKind.NamedImports:
                 case ts.SyntaxKind.NamedExports:
                     this.checkList((node as ts.ArrayLiteralExpression | ts.BindingPattern | ts.NamedImportsOrExports).elements,
-                                   node.end);
+                                   node.end, node);
```

Comment:
This doesn't work for the first few cases where array, object, imports and exports all fall through to the same case handler. I'd have to add `this.checkList(...)` for each case in `ArrayLiteral`, `ArrayBinding`, `ObjectBinding`, etc.